stacks/single-server/docker-compose.dev.yml

```diff
@@ -13,7 +13,7 @@ services:
       MYSQL_ROOT_PASSWORD: root
   openemr:
     restart: always
-    build: ../../docker-images/openemr/
+    build: ../../docker-images/openemr/5.0.0
```

Comment:
You removed the trailing slash, but I can't tell if it's needed or not.

Reply:
I'll run a build to make sure it's not

Reply:
Confirmed it's all good. Now if I would just stop making foolish mistakes in the ci config...